tools/upgrade-loader-or-syncer-to-dm.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Upgrade Loader or Syncer to Data Migration
+summary: This document introduces how to upgrade Loader or Syncer to Data Migration. 
+category: tools
+---
+
+# Upgrade Loader or Syncer to Data Migration
+
+This document introduces how to upgrade Loader or Syncer to DM (Data Migration).
+
+## Upgrade Loader to Data Migration
+
+Loader is used to load the full data that is dumped from [mydumper](../tools/mydumper.md) to TiDB.
+
+When the `task-mode` of the task DM executes is `full`, DM automatically uses `dumper` to export data and then uses `loader` to import the data.
+
+To upgrade Loader to DM, you only need to generate the corresponding task configuration file and use dmctl to execute the task.
```

Comment:
@csuzhangxc Could you please provide the concrete steps (like 1, 2, 3, ...) that users can directly follow? This one sentence is ambiguous. Users might still be not clear about how to go ahead.

Later, the dmctl link will be added to this doc after that PR is merged.

Reply:
1. deploy DM cluster
2. generate the corresponding task configuration file
3. use dmctl to `start-task`